Dockerfile

```diff
@@ -9,4 +9,6 @@ RUN npm install && \
 
 RUN grunt compile
 
+RUN rm -rf /code/node_modules/rx-amqplib/src
```

Comment:
It can be a solution but a module can well makes its "npm main file" in the src folder.